mslice/presenters/cut_widget_presenter.py

```diff
@@ -95,7 +95,7 @@ def _parse_input(self):
 
         norm_to_one = bool(self._cut_view.get_intensity_is_norm_to_one())
         width = self._cut_view.get_integration_width()
-        algo = self._main_presenter.get_cut_algorithm() if self._main_presenter is not None else 'Rebin'
+        algo = self._cut_view.get_cut_algorithm()
```

Comment:
```suggestion
        algo = ['Rebin', 'Integration'][self._cut_view.get_cut_algorithm()]
```

This is called by `_cut` to get parameters for the `Cut` algorithm, which expects a string in the `Algorithm` keyword argument. Whilst `_main_presenter.get_cut_algorithm` returned a string, `_cut_view.get_cut_algorithm` returns an index... and this causes the `Cut` algorithm to fail.

There is a `_name_to_index` but it is defined in the `CutWidget` class only so can't be used here...